evennia/accounts/accounts.py

```diff
@@ -1393,6 +1393,90 @@ def at_look(self, target=None, session=None, **kwargs):
             look_string = ("-" * 68) + "\n" + "".join(result) + "\n" + ("-" * 68)
             return look_string
 
+    def return_examine(self, viewer, *args, **kwargs):
```

Comment:
The common name for this kind of entity in used in the API would be `looker`; change to that everywhere. It should also pointed out in the docstring that this argument is not used in the  default implementation.